hub/cli/command.py

```diff
@@ -23,10 +23,17 @@ def cli(host, verbose):
     configure_logger(verbose)
 
 
+@click.command()
+def version(version=1.22):
+    """Get the version of the hub package"""
+    click.echo(f"you are using the hub version:{version}")
```

Comment:
Thanks for adding this - we definitely need to be able to retrieve the version from the CLI.
I would suggest storing a version in one central place (currently we keep it in setup.py but probably __version__ would be better) and then importing it in the CLI rather then setting it there. Also, it's better to return just the version or package name (Hub) + the version. Perhaps, the version should be just an argument to hub, not a separate command. In that case, we could invoke it in the following way: `hub --version`.

Compare `python --version`.

Reply:
Yeah tried to do but got some distribution error regarding some packages, so is it okay to store it in version.py and import as required.

Reply:
Added requested changes @mynameisvinn @haiyangdeperci